README.md

```diff
@@ -44,7 +44,7 @@ Workers are built in their own bin directories:
 * Client: `client/bin`
 
 ## Cleaning the project
-Run `./clean.sh` to delete all build files, including worker binaries and any intermediate files generated during the build process. Alternatively, run `git clean -xdf`.
+Run `git clean -xdf` to delete all build files, including worker binaries and any intermediate files generated during the build process.
```

Comment:
`git clean -xdf` makes me sad 🙁 Often does a lot of things you don't expect. If we want to recommend this then at least say `git clean -xdi` which shows you what files will be removed

Reply:
Ha! My mind is blown (`gti clean -xdi`). We used to have a `clean.sh` script but received requests to remove it 🤷‍♂️

Reply:
sure, changed!